tests/suites/test_suite_cipher.chacha20.data

```diff
@@ -1,7 +1,3 @@
-Decrypt empty buffer
-depends_on:MBEDTLS_CHACHA20_C
-dec_empty_buf:
```

Comment:
I don't understand why we can't have a `dec_empty_buf` test anymore. Should we instead parameterize the test to so it uses a cipher parameter?

It seems like we are reducing the intended test coverage here by removing this test. (I think we intended to have an empty buffer test that used CHACHA20.)

Reply:
BTW I don't think DJB likes HMAC_DRBG either, but CTR_DRBG and HMAC_DRBG are all we have. HMAC_DRBG builds on SHA2 which is pretty much indispensable anyway.

Reply:
I agree with Gilles that removing the test does not reduce the actual test coverage (which might indeed differ from the intended one, though I'm not sure our intentions here were very clear), so I think it would better be handled as a separate issue, in order to avoid feature creep in this one.

> BTW I don't think DJB likes HMAC_DRBG either, but CTR_DRBG and HMAC_DRBG are all we have. HMAC_DRBG builds on SHA2 which is pretty much indispensable anyway.

I assume DJB would prefer a DRBG based on ChaCha20, but as you say we don't have it (and I'm not sure one is standardized, so that could be risky to implement), and I think SHA-2 is the part of NIST Crypto that is also approved by DJB (used in Ed25519 for example), so as you say, since HMAC_DRBG builds on it... Btw, from my perspective the main benefit of a DRBG based on Chacha20 rather than HMAC would be speed, as otherwise both should have similar security properties.

Reply:
OK, noted as separate issue in #2014